easybuild/easyconfigs/a/Avogadro2/Avogadro2-1.97.0-linux-x86_64.eb

```diff
@@ -0,0 +1,29 @@
+easyblock = 'Binary'
+
+name = 'Avogadro2'
+version = '1.97.0'
+versionsuffix = '-linux-x86_64'
+
+homepage = 'https://two.avogadro.cc/index.html'
+description = """
+    Avogadro is an advanced molecule editor and visualizer designed for cross-platform 
+    use in computational chemistry, molecular modeling, bioinformatics, materials science, 
+    and related areas.
+"""
+
+toolchain = SYSTEM
+
+sources = [{
+    'source_urls': ['https://github.com/OpenChemistry/avogadroapp/releases/download/1.97.0'],
+    'filename': 'avogadro2',
```

Comment:
Please use the `%(version)s` template here so that only changing the version on top of the easyconfig file (or via `eb --try-software-version`) is sufficient.
Also, retain the version in the filename after downloading, otherwise we'll get a filename clash if we have easyconfigs for other version of `Avagadro2`.

To rename the installed file to `avogadro2`, you can use `postinstallcmds`:

```python
sources = [{
    'source_urls': ['https://github.com/OpenChemistry/avogadroapp/releases/download/%(version)s/'],
    'filename': '%(name)s-%(version)s.bin',
    'download_filename': 'Avogadro2-x86_64.AppImage',
}]
checksums = ['cc522dbdaa6e5386cb7101d382c5730f0d1ff71f2e70eb10f7e55f4ca052c89d']

postinstallcmds = ["mv %(installdir)s/%(name)s-%(version)s.bin %(installdir)s/avogadro2"]

```